newrelic-cli/tools/LICENSE.txt

```diff
@@ -0,0 +1,11 @@
+﻿
+Note: Include this file if including binaries you have the right to distribute. 
+Otherwise delete. this file.
+
+===DELETE ABOVE THIS LINE AND THIS LINE===

```

Comment:
Can we dynamically pull in the root LISENSE.md instead?

Reply:
Can just copy it in during the build.